.github/workflows/build_assets.yml

```
@@ -68,12 +69,12 @@ jobs:
         run: ${{matrix.CMD_BUILD}}
       - name: Upload a Build Artifact
         uses: actions/upload-artifact@v4
-        if: inputs.release == false
+        if: startsWith(github.ref, 'refs/tags/')
```

That's the old way. These days, there's a more straightforward var exposed in this context:
```suggestion
        if: github.ref_type == 'tag'
```

---

Thanks!